ServerlessLibraryAPI/ClientApp/src/components/DetailView/DetailPageContent.scss

```diff
@@ -1,12 +1,29 @@
 .pivot-item-container {
-  margin-left: 23px;
-  width: 100%;
+  padding-left: 23px;
+  background-color: #fbfbfb;
+}
+.wrapper {
```

Comment:
please give this a more specific name, e.g. scrollablePane-wrapper

Reply:
done